### PR TITLE
tests: split tests.go into tests.go+ selenium.go

### PR DESCRIPTION
**Pull request by hunchback**
plus some extra minor test related fixes

### CODE REVIEW

**Comment by hunchback** [comment]
Failed due to:

```
22:46:26.567 Caused: java.io.IOException: Remote call on JNLP4-connect connection from 149.14.153.26/149.14.153.26:6623 failed

```

**Comment by lebauce** [comment]
run tests

File: tests/selenium.go [hunk 1]
```diff
@@ -0,0 +1,598 @@
+/*
+ * Copyright (C) 2018 Skydive-Project Authors.
```
Comment (lebauce) [inline tests/selenium.go]: nit: you should probably use the same than `tests.go`

Comment (hunchback) [reply tests/selenium.go]: fixed

**Comment by lebauce** [comment]
run skydive-scale-tests

**Pull request merged**